src/projects/validator.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: layouts/project.liquid
+tags: ["Geodaten","Validierung","Datenqualität"]
+id: validator
+title: Prüftool für Geodaten
+abstract: Tool zur Schemavalidierung von Geodaten im GML-Format
+languages: ["JavaScript", "Python"]
+license: MIT License
+link: https://odis-berlin.de/projekte/xml-validator/tool
+repository: https://github.com/technologiestiftung/xml-schema-validator-for-berlins-geodata
+---
+
+# Berliner-Breitband-Portal
```

Comment:
Wrong title. But speaking of which: the title here can be removed altogether. We will be getting the page title from the front matter.